### PR TITLE
[iOS] - Adaptive Cards visual Improvements and fixes

### PR DESCRIPTION
**Pull request by GabrielMedAlv**
Visual improvements requested by design team:

Increased height of choice set cell to 44 px
Additional separation between choice icon and text (8 px)

Before:

Simulator Screenshot - iPhone 14 - 2023-09-29 at 15 21 39

After:

Simulator Screenshot - iPhone 14 - 2023-10-04 at 16 19 56

### CODE REVIEW

**Review by ardlank — APPROVED** [review]

**Review by karthikgopal36 — APPROVED** [review]

**Review by jwoo-msft — APPROVED** [review]

**Review by MengliElmendorf137 — APPROVED** [review]

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm [hunk 1]
```diff
@@ -19,7 +19,8 @@
 NSString *checkedRadioButtonReuseID = @"checked-radiobutton";
 NSString *uncheckedRadioButtonReuseID = @"unchecked-radiobutton";
 
-const CGFloat padding = 2.0f;
+const CGFloat padding = 8.0f;
```
Comment (VikrantSingh01) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:22]: Do we need to add this new expectation in extenal facing docs @jwoo-msft  ?

Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:22]: @VikrantSingh01 
We could add a note about that to the release notes.

Comment (VikrantSingh01) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:22]: @GabrielMedAlv : fyi ^

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm [hunk 2]
```diff
@@ -269,8 +270,10 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
                               attributes:@{NSFontAttributeName : cell.textLabel.font}
                                  context:nil]
             .size;
-
-    return labelStringSize.height + _spacing;
+    
+    CGFloat rowHeight = labelStringSize.height + _spacing;
+    
+    return (rowHeight < minimumRowHeight) ? minimumRowHeight : rowHeight;
```
Comment (VikrantSingh01) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:276]: Lets test the flow. Do we have any UT to validate this change ?

**Review by VikrantSingh01 — APPROVED** [review]
Pls. take care of below two comments.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm (hunk 2 above)
Comment (GabrielMedAlv) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:276]: There are no tests for ACRChoiceSetView currently. I will work to add unit tests for some components in another PR.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm (hunk 1 above)
Comment (GabrielMedAlv) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:22]: I plan to add these as part of release notes:

- Increased height of choice set cell to 44 pixels
- Increased separation between choice icon and text to 8 pixels.

**Pull request merged**